Directory.Packages.props

```diff
@@ -14,9 +14,12 @@
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageVersion Include="NodaTime" Version="3.1.9" />
     <PackageVersion Include="NSwag.Core.Yaml" Version="14.0.0-preview011" />
+    <PackageVersion Include="Parlot" Version="0.0.25.0" />
```

Comment:
Source generators do not support transient dependencies: https://github.com/dotnet/sdk/issues/17775
Because of that need to explicitly install some transient packages.
For example, it'll be used here to include `Parlot` into NuGet package https://github.com/RicoSuter/NJsonSchema/pull/1726/files#diff-ead3877feb09b2619c61d0feb8b85e0c85c627756febe0dda0829dafe13d5325R39